deepspeed/utils/config.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+# DeepSpeed Team
+
+from deepspeed.runtime.config_utils import get_scalar_param, DeepSpeedConfigObject
+from deepspeed.utils.constants import *
+
+
+class DeepSpeedThroughputTimerConfig(DeepSpeedConfigObject):
```

Comment:
This should use the `DeepSpeedConfigModel` rather than `DeepSpeedConfigObject` to take advantage of config validation via pydantic: https://github.com/microsoft/DeepSpeed/blob/4d4c99d9e51afa01a8f4fc7d59a0bc09128abf3e/deepspeed/runtime/config_utils.py#L16

Reply:
@mrwyattii, the requested changes have been made. Thanks.